### PR TITLE
Fix grammar and broken image link

### PR DESCRIPTION
**Pull request by Keavon**

### CODE REVIEW

File: en-US/features/authentication.md [hunk 1]
```diff
@@ -175,18 +176,17 @@ This option allows Gogs to log in to your SMTP host as a Gogs user. To configure
...
+-  Add an IPA group for `gogs_users`:
 ```
   ipa group-add --desc="Gogs Users" gogs_users
 ```
--  Note! if you get error about ipa credentials please run kinit admin and give your admin accound password.
-
--  Now login to the gogs as an Admin, click on “Authentication” under Admin Panel. Then click New LDAP Source and fill in the details, changing all where appropriate to your own domain as photo below:
+-  Note! If you get an error about IPA credentials, run kinit admin and give your admin accound password.
 
-![](/docs/images/Freeipa-Gogs.png)
```
Comment (Unknwon) [inline en-US/features/authentication.md]: This URL works perfectly fine in terms of the actual hosting of the site, not the Markdown: https://gogs.io/docs/features/authentication

Comment (Keavon) [reply en-US/features/authentication.md]: Would it be possible to fix the overall problem, then, so the site is built with the same relative path structure as the Markdown?

Comment (Unknwon) [reply en-US/features/authentication.md]: This documentation set is meant to work with https://peachdocs.org/. You can always preview at real time.

**Comment by Unknwon** [comment]
Thank you!

**Pull request merged**